hindi/Bhagat_Singh/bhagat_singh.md

```diff
@@ -0,0 +1,13 @@
+Bhagat Singh aka Shaheed Bhagat Singh (28 September 1907 – 23 March 1931) was an [Indian](https://simple.wikipedia.org/wiki/India) socialist and a revolutionary. He is considered to be one of the most influential revolutionaries of the [Indian Independence Movement](https://simple.wikipedia.org/wiki/Indian_independence_movement).
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.